arb-gateway/README.md

```diff
@@ -27,7 +27,7 @@ wrngler login
 wrangler secret put L1_PROVIDER_URL
 wrangler secret put L2_PROVIDER_URL
 wrangler secret put L2_ROLLUP
-yarn deploy
+yarn deploy --env sepolia
```

Comment:
Didn't you decide to use environment variables for this?

Reply:
Yes. it uses the env variable specified on wrangler.toml and that's how to pass the environment

https://github.com/ensdomains/evmgateway/pull/26/files#diff-17f03ef4a17652e904f64bff4ddeb0e9a95bbcf3d0f6f603f3befd4b5fc6d368R14